py3-jupyter-telemetry.yaml

```
@@ -0,0 +1,41 @@
+# Generated from https://pypi.org/project/jupyter-telemetry/
+package:
+  name: py3-jupyter-telemetry
+  version: 0.1.0
+  epoch: 0
+  description: Jupyter telemetry library
+  copyright:
+    - license: BSD
```

Re: https://github.com/jupyter/telemetry/blob/master/LICENSE

```suggestion
    - license: BSD-3-Clause
```